app/controllers/concerns/twitter_concern.rb

```diff
@@ -15,6 +15,8 @@ def twitter_client
     Twitty::Facade.new do |config|
       config.consumer_key = ENV.fetch('TWITTER_CONSUMER_KEY', nil)
       config.consumer_secret = ENV.fetch('TWITTER_CONSUMER_SECRET', nil)
+      config.access_token = ENV.fetch('TWITTER_ACCESS_TOKEN', nil)
```

Comment:
This doesn't need to be fetched from the environment variable. These values are unique on inbox level and are obtained at the end of auth flow